libs/components/tabs/src/lib/modules/tabs/tabset.component.spec.ts

```diff
@@ -1814,6 +1814,8 @@ describe('Tabset component', () => {
 
       expect(replaceStateSpy).toHaveBeenCalledOnceWith(
         '/?foobar-active-tab=design-guidelines',
+        '',
+        jasmine.any(Object),
```

Comment:
Angular's router calls the same location replace method, but it passes two additional parameters.